tests/test_config.yml

```diff
@@ -103,6 +103,7 @@ expected_dbt_ls_results:
       - *orders
   exclude:
     customers:
+      - *revenue_by_daily_cohort
```

Comment:
Is this change related/caused by the change in _utils.py?

Reply:
Yes, it's because `revenue_by_daily_cohort` is an `analysis` resource type, but analyses are ignored by `dbt ls` unless the `--resource-type analysis` flag is explicitly given.  

Before the change in _utils.py, the test was something similar to:
- `dbt ls --exclude customers`

After the change, it is similar to:
- `dbt ls --resource-type model --resource-type seed --resource-type snapshot --resource-type analysis`

Since after the change `--resource-type analysis` is given, `revenue_by_daily_cohort` will now be in the results.

Reply:
Thanks for explaining! That makes sense.